app/Http/Controllers/Widgets/WidgetController.php

```diff
@@ -60,14 +60,11 @@ public function getSettingsView(Request $request)
     public function __invoke(Request $request)
     {
         $this->show_settings = (bool) $request->get('settings');
+        $settings = $this->getSettings();
 
         if ($this->show_settings) {
             $view = $this->getSettingsView($request);
-        }
```

Comment:
I think the intention here was to avoid fetching settings when they are not needed.
However, it doesn't look like the code succeeds at that, perhaps it was changed after the fact.

Reply:
Yeah I thought the same first, but seems it just loads it anyway